packages/atomic/src/components/insight/atomic-insight-generated-answer/atomic-insight-generated-answer.tsx

```diff
@@ -101,6 +105,12 @@ export class AtomicInsightGeneratedAnswer
    */
   @Prop() collapsible?: boolean;
 
+  /**
+   * The specified height in rem to allow the answer to show before collapsing.
+   *
+   */
+  @Prop() maxCollapsedHeight = this.DEFAULT_COLLAPSED_HEIGHT;
```

Comment:
Why is it a "max" collapsed height?

Shouldn't the widget always have the same height by default in order to prevent layout shifts?

Reply:
it is the max height of the answer that we want to reach before collapsing